### ISSUE TITLE
Open all links in system browser (HTML Webview)

### ISSUE THREAD

**Issue by cpenner-va** [folivoraAI/BetterTouchTool#2633]
**Description of feature request:**
Loving the new webview overlay! I've been using it to build an autocomplete search box for bookmarks and such; when I make a selection I want it to open the link in chrome. I'm currently doing this by using `window.location.href = 'https://example.com'` and setting the "open URLs with prefix in system browser" to `http`. This works great; but I also built a slack integration into the autocomplete which opens up slack to a given channel; to do so I navigate to `slack://channel?id=<channelID>&team=<teamID>`; when this opens in the webview it doesn't work; if I switch my URL prefix to `slack` then it works, but now my http links open in the webview.

Basically I'd love to be able to just open ALL links in the system browser OR be able to set multiple prefixes (or maybe a regex?). I've tried selecting the checkbox for "open URLS with Prefix" and setting the prefix to an empty string, but it just disables the feature entirely

**Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):**
MacBook

Thanks for all your work on this! Love the tool!

**Comment by fifafu** [comment]
The newest alpha (v2.472) allows to enter a comma separated list  e.g. http,slack
